src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -84,109 +52,49 @@ class SidebarScreen extends Component {
             isCreateMenuActive: !state.isCreateMenuActive,
         }));
     }
 
-    /**
-     * Method called when we want to show the navigationMenu,
-     * will not do anything if it already open
-     * Only changes navigationMenu state on smaller screens (e.g. Mobile and mWeb)
-     */
-    showNavigationMenu() {
-        if (this.props.isSidebarShown) {
-            return;
-        }
-
-        showSidebar();
-    }
-
-    /**
-     * Animates the navigationMenu in and out.
-     *
-     * @param {Boolean} navigationMenuIsShown
-     */
-    animateNavigationMenu(navigationMenuIsShown) {
-        Animated.timing(this.animation, {
-            toValue: navigationMenuIsShown ? 0 : 1,
-            duration: 300,
-            useNativeDriver: true,
-        }).start();
-    }
-
     /**
      * Method called when a pinned chat is selected.
      */
-    recordTimerAndHideSidebar() {
+    startTimer() {
         Timing.start(CONST.TIMING.SWITCH_REPORT);
-
-        if (!this.props.isSmallScreenWidth) {
-            return;
-        }
-
-        hideSidebar();
     }
 
     render() {
-        const sidebarWidth = this.props.isSmallScreenWidth
-            ? this.props.windowWidth
-            : variables.sideBarWidth;
-        const translateXValue = this.animation.interpolate({
-            inputRange: [0, 1],
-            outputRange: [-sidebarWidth, 0],
-        });
         return (
-            <Animated.View style={[
-                getNavigationMenuStyle(
-                    this.props.windowWidth,
-                    this.props.isSmallScreenWidth,
-                ),
-                {
-                    transform: [
-                        {translateX: translateXValue},
-                    ],
-                }]}
+            <ScreenWrapper
+                includePaddingBottom={false}
+                style={[styles.sidebar]}
             >
-                <ScreenWrapper
-                    includePaddingBottom={false}
-                >
-                    {insets => (
-                        <>
-                            <View style={[styles.flex1, styles.sidebar]}>
-                                <SidebarLinks
-                                    onLinkClick={this.recordTimerAndHideSidebar}
-                                    insets={insets}
-                                    onAvatarClick={this.navigateToSettings}
-                                />
-                                <FAB
-                                    isActive={this.state.isCreateMenuActive}
-                                    onPress={this.toggleCreateMenu}
-                                />
-                            </View>
-                            <CreateMenu
-                                onClose={this.toggleCreateMenu}
-                                isVisible={this.state.isCreateMenuActive}
-                                onItemSelected={this.onCreateMenuItemSelected}
+                {insets => (
```

Comment:
if this is just to avoid prop drilling, you can prob set up a context provider and all the children should have access to the insets

Reply:
There is a context consumer inside the `ScreenWrapper` and a provider higher up, but I might not be understanding what specific change you are suggesting.

https://github.com/Expensify/Expensify.cash/blob/98b67a49ec6405978013e113d73913c83b0a109a/src/App.js#L6-L11
https://github.com/Expensify/Expensify.cash/blob/98b67a49ec6405978013e113d73913c83b0a109a/src/components/ScreenWrapper.js#L29-L30